rsts/deployment/cluster_config/customizable_resources.rst

```diff
@@ -73,6 +77,6 @@ For example:
 
 Flytekit
 ^^^^^^^^
-For convenience, add a flyte-cli wrapper for updating your new attribute. See `this PR <https://github.com/flyteorg/flytekit/pull/174>`__ for the full required set of changes.
+For convenience, add a flyte-cli wrapper for updating your new attribute. See `this PR <https://github.com/flyteorg/flytekit/pull/174>`__ for the entire set of changes required.
```

Comment:
@evalsocket, do we have to replace this section with FlyteCTL? Can you help with the content?